docs/concepts.md

```diff
@@ -24,8 +29,14 @@ Such key activities include
  MergeOnRead storage type of dataset
  * `COMPACTIONS` - Background activity to reconcile differential data structures within Hudi e.g: moving updates from row based log files to columnar formats.
 
```

Comment:
Should we talk about other actions like Cleaning here ? We have some documentation in the Configuration section for Cleaner but not much anywhere else.

Reply:
Cleaning is actually mentioned above.. I added rollback and savepoints